arlo-client/src/App.tsx

```diff
@@ -31,7 +31,7 @@ const App: React.FC = () => {
               path="/election/:electionId/board/:token"
               component={DataEntry}
             />
-            <Route path="/election/:electionId" component={Audit} />
+            <Route path="/election/:electionId/:stage?" component={Audit} />
```

Comment:
Why `/:stage?`? Is that question mark required?

Reply:
> Why `/:stage?`? Is that question mark required?

@eventualbuddha That makes that param optional. It was needed at one point in the development of this feature, but with how I ended up using the react router hooks, it isn't needed now. Removed it!